README.md

```diff
@@ -227,21 +300,43 @@ void helloWorld(ExpressHttpRequest request) async {
 }
 ```
 
-## HTTPS functions details
+## HTTPS Functions Details
 
-Firebase uses Expressjs web framework for HTTPS functions with `body-parser`
-middleware enabled by default ([documentation][body_parser_docs]).
+Firebase uses the Express.js web framework for HTTPS functions with `body-parser`
+middleware enabled by default ([documentation](https://firebase.google.com/docs/functions/http-events#read_values_from_the_request)).
 
 The `ExpressHttpRequest` exposed by this library extends standard `dart:io`
 `HttpRequest` interface, which means it is also a stream of bytes. However
 if `body-parser` middleware already decoded request body then listening for
 data on the request would hang since it's already been consumed. Use
 `ExpressHttpRequest.body` field to get decoded request body in this case.
 
-[body_parser_docs]: https://firebase.google.com/docs/functions/http-events#read_values_from_the_request
 
-## Features and bugs
+## Features and Bugs
 
-Please file feature requests and bugs at the [issue tracker][tracker].
+Please file feature requests and bugs at the [issue tracker](https://github.com/pulyaevskiy/firebase-functions-interop/issues/new).
 
-[tracker]: https://github.com/pulyaevskiy/firebase-functions-interop/issues/new
+## Testing
```

Comment:
There is a separate `DEVELOPMENT.md` which covers these steps as well.

I think it would make more sense to move this section to `DEVELOPMENT.md` and only leave a reference to that file here. WDYT?

Reply:
Oh my! I totally skipped over that and had to reverse the whole testing process. I'll just add a link to DEVELOPMENT.md here instead!